packages/datagrid/src/components/DefaultCellRenderer/DefaultCellRenderer.component.js

```diff
@@ -12,13 +13,20 @@ import theme from './DefaultCell.scss';
 
 export const CELL_RENDERER_COMPONENT = 'cellRenderer';
 
+function convertValue(value) {
+	if (!value.toJS) {
+		return value;
+	}
+
+	return value.toJS();
+}
+
+const memoizedConvertValue = memoize(convertValue);
```

Comment:
You never clear the cache, I think you you use a memoize-one solution.
Here it's a functional component, you can't attach a memoized function by instance, but there is the React.useMemo hook 🎉

Reply:
We have decided that use react.memo isn't interesting. ag-grid use a virtualization so the row is destroy or add when the user scrolls. So, we lost the benefit of react.memo when the row is destroyed.